README_TASK.md

```
@@ -62,7 +75,7 @@
     * Int16, UInt16, Int32, UInt32, Int64, UInt64.
     * Single, Double.
     * Guid (16 bytes).
-  * [ ] Write/Read string (UInt16 length prefix, UTF-8 encoded).
+  * [x] Write/Read string (UInt16 length prefix, UTF-8 encoded).
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)